app/tests.bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
This file is added, why so? (if it can't be thought it, it's likely it's not needed and should be added to the `.gitignore` file.